### ISSUE TITLE
Re-enable `crayon.enabled = TRUE`

### ISSUE THREAD

**Issue by mine-cetinkaya-rundel** [hadley/r4ds#1027]
With the conversion to Quarto I've commented out `crayon.enabled = TRUE` in `_common.R`. We need to make sure to re-enable this once the issue is resolved.

**Comment by mine-cetinkaya-rundel** [comment]
Do we still need this?

**Comment by hadley** [comment]
Yeah, we should eventually figure out how to fix this, so (e.g.) https://r4ds.hadley.nz/data-transform#prerequisites gets the nice colouring.
